src/pipeline.cc

```diff
@@ -76,18 +77,23 @@ using sharp::FreeCallback;
 using sharp::CalculateCrop;
 using sharp::counterProcess;
 using sharp::counterQueue;
+using sharp::GetBooleanOperation;
 
 class PipelineWorker : public AsyncWorker {
  public:
   PipelineWorker(Callback *callback, PipelineBaton *baton, Callback *queueListener,
-    const Local<Object> &bufferIn, const Local<Object> &overlayBufferIn) :
+    const Local<Object> &bufferIn, const Local<Object> &overlayBufferIn,
```

Comment:
How about we make this a `std::vector<Local<Object>>` of buffers that need to be involved in a call to `SaveToPersistent` and `GetFromPersistent`?